typescript/infra/config/environments/testnet3/agent.ts

```diff
@@ -95,10 +107,49 @@ const releaseCandidate: RootAgentConfig = {
     connectionType: AgentConnectionType.HttpFallback,
     docker: {
       repo,
-      tag: '2deb9b8-20230602-205342',
+      tag: '29b6bd2-20230620-134737',
     },
-    whitelist: releaseCandidateHelloworldMatchingList,
-    gasPaymentEnforcement,
+    whitelist: [
+      ...releaseCandidateHelloworldMatchingList,
+      // Whitelist all traffic to solanadevnet and solanadevnet1
+      {
+        originDomain: '*',
+        senderAddress: '*',
+        destinationDomain: [
+          getDomainId(chainMetadata.solanadevnet),
+          getDomainId(chainMetadata.solanadevnet1),
+        ],
+        recipientAddress: '*',
+      },
+      // Whitelist all traffic from solanadevnet and solanadevnet1 to fuji
+      {
+        originDomain: [
+          getDomainId(chainMetadata.solanadevnet),
+          getDomainId(chainMetadata.solanadevnet1),
+        ],
+        senderAddress: '*',
+        destinationDomain: [getDomainId(chainMetadata.fuji)],
+        recipientAddress: '*',
+      },
+    ],
+    gasPaymentEnforcement: [
+      // Don't require gas payments from solanadevnet or solanadevnet1
+      {
+        type: GasPaymentEnforcementPolicyType.None,
+        matchingList: [
+          {
+            originDomain: [
+              getDomainId(chainMetadata.solanadevnet),
+              getDomainId(chainMetadata.solanadevnet1),
+            ],
+            senderAddress: '*',
+            destinationDomain: [getDomainId(chainMetadata.fuji)],
+            recipientAddress: '*',
+          },
+        ],
+      },
+      ...gasPaymentEnforcement,
+    ],
```

Comment:
nice
I wonder if this should be a GCP secret?

Reply:
What would be the benefit? Less obvious what the configurations are to protect from griefing? would require some decently large changes